src/resolve/resolved-dns-server.c

```diff
@@ -719,9 +719,9 @@ DnsServer *manager_set_dns_server(Manager *m, DnsServer *s) {
                 return s;
 
         if (s)
-                log_info("Switching to %s DNS server %s.",
-                         dns_server_type_to_string(s->type),
-                         dns_server_string(s));
+                log_debug("Switching to %s DNS server %s.",
+                          dns_server_type_to_string(s->type),
+                          dns_server_string(s));
```

Comment:
No, I don't have an opinion, because I haven't had time to look into this properly. It's possible that recently those messages have been less frequent then before, probably because of that patch. Like I said, I think this is OK to merge.